talktorials/11_online_apis/11a_obtaining_structures.ipynb

```diff
@@ -0,0 +1,2943 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Talktorial 11 (part A)\n",
+    "\n",
+    "# CADD web services that can be used via a Python API\n",
+    "\n",
+    "__Developed at AG Volkamer, Charité__\n",
+    "\n",
+    "Dr. Jaime Rodríguez-Guerra\n",
+    "\n",
+    "## Aim of this talktorial\n",
+    "\n",
+    "Web services are a convenient way of using software because it frees the user from any installation hassles. A web UI is usually available for easy usage, at the cost of losing the possibility to automate a workflow. Fortunately, the number of web services that provide an API for automated access has been increasing. Some examples in the field of Computer Aided Drug Design include:\n",
+    "\n",
+    "- ChEMBL\n",
+    "- RCSB PDB\n",
+    "- KLIFS\n",
+    "- Proteins.plus\n",
+    "- SwissDock\n",
+    "\n",
+    "In this notebook, you will learn how to programmatically use online web-services from Python, always in the context of drug design. The end goal will be to build a full pipeline that exclusively relies on web-services, without (almost) any local execution!\n",
+    "\n",
+    "__Note__: For simplicity, the full lesson will be divided in three notebooks:\n",
+    "\n",
+    "- 11a. Querying KLIFS & PubChem for potential kinase inhibitors\n",
+    "- 11b. Docking the candidates against the target obtained in 11a\n",
+    "- 11c. Assessing the results and comparing against known data\n",
+    "\n",
+    "## Learning goals\n",
+    "\n",
+    "### Theory\n",
+    "\n",
+    "- Types of programmatic access\n",
+    "- Description of the pipeline and the involved webservices\n",
+    "\n",
+    "### Practical\n",
+    "\n",
+    "- Building the pipeline\n",
+    "- Case study: EGFR inhibitors\n",
+    "\n",
+    "### Discussion\n",
+    "\n",
+    "See below.\n",
+    "\n",
+    "### Quiz\n",
+    "\n",
+    "See below.\n",
+    "\n",
```

Comment:
In the previous talktorials, we did not mention the Dissussion/Quiz sections in the "Learning goals" section.